docs/source/cloud/architecture.rst

```diff
@@ -0,0 +1,66 @@
+===========================================
+The Registry API Service Cloud Architecture
+===========================================
+
+On-Prem vs. Cloud
+*****************
+
+In a typical on-premise deployment, the PDS Registry Application is composed, server-side of
+a ReSTful API Server which queries Opensearch indices in response to
+user-submitted search requests - both components of which are highlighted
+below:
+
+.. image:: /_static/images/registry-onprem.png
+
+Given the stateless nature of the API service, scaling is a matter of placing
+a load balancer in front of a number of dedicated hosts which run copies of
+the service. While relatively minor in complexity, this strategy nonetheless
+constitutes a significant administrative overhead and potentially wastes
+resources that must be running in anticipation of peak loads that run largely
+idle for the remaining time. These issues are addressed by two of the key
+advantages of cloud deployments.
+
+The API service's cloud architecture is given in the diagram below. While
+seemingly more complex than an on-prem deployment, the utilization of
+AWS services simplifies administration, streamlines scaling with little
+idle resources and better secures management of sensitive data such
+as Elasticsearch credentials.
+
+.. image:: /_static/images/registry_cloud.png
+
+The centerpiece of the cloud architecture is the use of AWS Fargate, a
+flavor of the Elastic Container Service (ECS) which supports the execution
+of containerized applications (tasks) without requiring explicit
+provisioning of compute resources. The images for these application
+containers are placed in and pulled from the Elastic Container Registry
+(ECR). Each service instance connects to Elasticsearch, with the
+necessary credentials injected into each container by the Secrets
+Manager. Other non-sensitive run-time configuration is provided by the
+Systems Manager Parameter Store.
+
+Amazon's content delivery network, Cloudfront, is placed in front of
+the Elastic Load Balancer (ELB). Cloudfront provides high-speed, low-latency
```

Comment:
I feel it is better to reword "Cloudfront provides high-speed, low-latency access while also ensuring unintention or malicious bulk downloading of data is avoided". 

Suggestion (may be not the best suggestion) : "Cloudfront provides high-speed, low-latency access, while also ensuring the avoidance of  **unintentional** or malicious bulk downloading of data".

Reply:
What about "Cloudfront provides high-speed, low-latency
access while also preventing from unintentional or malicious bulk download
of data" ?

Reply:
Your suggestion is good!